ts/screens/preferences/OldServicesHomeScreen.tsx

```diff
@@ -143,19 +147,23 @@ export const getAllSections = createSelector(
     const orgfiscalCodes = Object.keys(services.byOrgFiscalCode);
     return orgfiscalCodes
       .map(fiscalCode => {
-        const title = organizations[fiscalCode] || fiscalCode;
```

Comment:
you could remove **organizationsSelector**  and use [organizationNamesByFiscalCodeSelector](https://github.com/teamdigitale/io-app/blob/d5ed3c0b5beebe81b31794f05cdc2e3622715319/ts/store/reducers/entities/organizations/organizationsByFiscalCodeReducer.ts)

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1214/commits/79d09b50b604bdbd71b7e0e7db0433977eadca36